docs/samples/Microsoft.ML.Samples/Trainers.cs

```diff
@@ -74,5 +76,122 @@ public static void SdcaRegression()
             Console.WriteLine($"RMS - {metrics.Rms}");  // 4.924493
             Console.WriteLine($"RSquared - {metrics.RSquared}");  // 0.565467
         }
+
+        public static void FastTreeRegression()
+        {
+            // Downloading a regression dataset from github.com/dotnet/machinelearning
+            // this will create a housing.txt file in the filsystem this code will run
+            // you can open the file to see the data. 
+            string dataFile = SamplesUtils.DatasetUtils.DownloadHousingRegressionDataset();
+
+            // Creating the ML.Net IHostEnvironment object, needed for the pipeline
+            var env = new LocalEnvironment(seed: 0);
+
+            // Creating the ML context, based on the task performed.
+            var regressionContext = new RegressionContext(env);
+
+            // Creating a data reader, based on the format of the data
+            var reader = TextLoader.CreateReader(env, c => (
+                        label: c.LoadFloat(0),
+                        features: c.LoadFloat(1, 6)
+                    ),
+                separator: '\t', hasHeader: true);
+
+            // Read the data, and leave 10% out, so we can use them for testing
+            var data = reader.Read(new MultiFileSource(dataFile));
+            var (trainData, testData) = regressionContext.TrainTestSplit(data, testFraction: 0.1);
+
+            // The predictor that gets produced out of training
+            FastTreeRegressionPredictor pred = null;
+
+            // Create the estimator
+            var learningPipeline = reader.MakeNewEstimator()
+                .Append(r => (r.label, score: regressionContext.Trainers.FastTree(
+                                            r.label,
+                                            r.features,
+                                        onFit: p => pred = p)
+                                )
+                        );
+
+            // Fit this pipeline to the training data
+            var model = learningPipeline.Fit(trainData);
+
+            // Check the weights that the model learned
+            VBuffer<float> weights = default;
+            pred.GetFeatureWeights(ref weights);
+
+            Console.WriteLine($"weight 0 - {weights.Values[0]}");
```

Comment:
>weights.Values[0] [](start = 44, length = 17)

This is not ALWAYS the first weight, because the `VBuffer` can be sparse. 
If you want to densify, you can do this:
```csharp
float[] allWeights = weights.DenseValues();
Console.WriteLine($"weight 0 - {allWeights[0]}");
Console.WriteLine($"weight 0 - {allWeights[0]}");
```